### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by wirekurosastak** [danielkrupinski/Osiris#3927] [Q&A]
![image](https://user-images.githubusercontent.com/75095332/211151393-547fca9c-afb2-4c2e-aeff-ad77662a9a7c.png)

**Comment by wirekurosastak** [comment]
@csgofever

**Reply by Xsintashi** [reply]
E0312: no suitable user-defined conversion from "Entity" to "TraceFilter" exists

You can't make trace from entity.

**Reply by wirekurosastak** [reply]
I see but how do I fix?

**Reply by Xsintashi** [reply]
I guess this code is outdated in this osiris version, cuz i had used this code and barely worked + fps dropped cuz of for loop.
mbe just use localPlayer instead of localPlayer.get()
